boa/src/builtins/json.rs

```diff
@@ -23,20 +22,39 @@ pub fn parse(_: &Value, args: &[Value], _: &mut Interpreter) -> ResultValue {
 /// Process a Javascript object into a JSON string
 pub fn stringify(_: &Value, args: &[Value], _: &mut Interpreter) -> ResultValue {
     let obj = args.get(0).expect("cannot get argument for JSON.stringify");
-    let json = obj.to_json();
-    Ok(to_value(to_string_pretty(&json).expect("")))
+    let json: String = obj.to_json().to_string();
+    Ok(to_value(json))
 }
 
 /// Create a new `JSON` object
 pub fn create_constructor(global: &Value) -> Value {
-    let mut json = Object::default();
-    json.kind = ObjectKind::Ordinary;
+    let json = ValueData::new_obj(Some(global));
 
-    let prototype = ValueData::new_obj(Some(global));
-    make_builtin_fn!(parse, named "parse", with length 2, of prototype);
-    make_builtin_fn!(stringify, named "stringify", with length 3, of prototype);
+    make_builtin_fn!(parse, named "parse", with length 2, of json);
+    make_builtin_fn!(stringify, named "stringify", with length 3, of json);
 
-    let json_value = to_value(json);
-    json_value.set_field_slice(PROTOTYPE, prototype);
-    json_value
+    to_value(json)
+}
+
+#[cfg(test)]
+mod tests {
+    use crate::exec::Executor;
+    use crate::forward;
+    use crate::realm::Realm;
```

Comment:
I think it's more clear if we use this syntax for imports. It makes it clear all we import from the crate:
```suggestion
    use crate::{
    	exec::Executor,
    	forward,
    	realm::Realm,
    };

```